nuxeo-core/nuxeo-core-bulk/src/test/resources/OSGI-INF/bulk-stream-configuration-tests.xml

```diff
@@ -6,7 +6,7 @@
   </extension>
 
   <extension target="org.nuxeo.runtime.stream.service" point="logConfig">
-    <logConfig name="bulk" type="chronicle" />
+    <logConfig name="bulk" type="${nuxeo.test.stream.type}" />
```

Comment:
I don't think we have a convention on that, but for storage and directory there is no `type` suffix, it's only `nuxeo.test.core` and `nuxeo.test.directory` for example